test/integration/015_cli_invocation_tests/test_cli_invocation.py

```diff
@@ -173,3 +173,104 @@ def _run_simple_dbt_commands(self, project_dir):
         # In case of 'dbt clean' also test that the clean-targets directories were deleted.
         for target in self.config.clean_targets:
             assert not os.path.isdir(target)
+
+
+class TestCLIInvocationWithProfilesAndProjectDir(ModelCopyingIntegrationTest):
+
+    def setUp(self):
+        super().setUp()
+
+        self.run_sql(f"DROP SCHEMA IF EXISTS {self.custom_schema} CASCADE;")
+        self.run_sql(f"CREATE SCHEMA {self.custom_schema};")
+
+        # the test framework will remove this in teardown for us.
+        if not os.path.exists('./dbt-profile'):
+            os.makedirs('./dbt-profile')
+
+        with open("./dbt-profile/profiles.yml", 'w') as f:
+            yaml.safe_dump(self.custom_profile_config(), f, default_flow_style=True)
+
+        self.run_sql_file("seed_custom.sql")
+
+    def tearDown(self):
+        self.run_sql(f"DROP SCHEMA IF EXISTS {self.custom_schema} CASCADE;")
+        super().tearDown()
+
+    def custom_profile_config(self):
+        return {
+            'config': {
+                'send_anonymous_usage_stats': False
+            },
+            'test': {
+                'outputs': {
+                    'default': {
+                        'type': 'postgres',
+                        'threads': 1,
+                        'host': self.database_host,
+                        'port': 5432,
+                        'user': 'root',
+                        'pass': 'password',
+                        'dbname': 'dbt',
+                        'schema': self.custom_schema
+                    },
+                },
+                'target': 'default',
+            }
+        }
+
+    @property
+    def schema(self):
+        return "test_cli_invocation_015"
+
+    @property
+    def custom_schema(self):
+        return "{}_custom".format(self.unique_schema())
+
+    @property
+    def models(self):
+        return "models"
+
+    @use_profile('postgres')
+    def test_postgres_dbt_commands_with_relative_dir_as_project_dir(self):
+        workdir = os.getcwd()
+        with tempfile.TemporaryDirectory() as profiles_dir:
```

Comment:
I find this a bit ugly, but since the `profiles-dir` is needed for the `setUp` I decided on moving it here.